### ISSUE TITLE
Hisuian Zorua? (This is mainly a me thing lol)

### ISSUE THREAD

**Issue by CommandBlockGuru** [mFireworks/Cobblemon-Outbreaks#1]
I moved from Better Mass Outbreaks to this datapack because the other one didn't level scale the mons to my level so they were all nearly one-shotted and weren't a good way to get exp. I really really want a shiny Hisuian Zorua, it's one of my favorite shiny mons ever, and I was really sad to see it missing from the list when I was going through it. It'd be lovely for it to be added. I'll try just moving the file from Better Outbreaks to this one's outbreaks folder and hope that works for me for now but it'd be really nice to have the chance on getting the shiny I really want not at like- level 9.

**Comment by mFireworks** [comment]
Yeah, I haven't updated the datapack to include the new 1.5 spawns, just haven't gotten around to it.

Not sure when I'll get around to updating it, it's a bit low on my priority list atm, but here's at least the hisuian zorua info for it. That'll at least solve your use case for now.

[hisuian-zorua_outbreak.json](https://github.com/user-attachments/files/16404824/hisuian-zorua_outbreak.json)

**Comment by CommandBlockGuru** [comment]
Thank you so much, I can't wait to shiny hunt for it :]

**Comment by mFireworks** [comment]
With the completion of #2, this is now included in the main pack.
